README.md

```diff
@@ -1,2 +1,121 @@
 # Big Ol' Chungus
 
+
+## Table of Contents
+
+  * [Installation](#setup)
+    * [Automatic](#automatic) (**Ubuntu only, recommended.**)
+    * [Building](#building) 
+  * [Running](#running)
+    * [Manual](#manual)
+    * [Systemd](#systemd) (**recommended**)
+  * [Issues](#issues)
+  * [Troubleshooting](#troubleshooting)
+
+## Installation
+
+#### Automatic
+
+Currently automatic installation is supported only in Ubuntu, and only for running via `systemd`.  If you do not want to
+use systemd, or you are not on ubuntu, perform the [manual](#manual) installation.
+
+If you are on ubuntu, and you do wish to use systemd, the following should be sufficient for downloading and installing 
+the latest version of BigOlChungus.
+
+```bash
+wget https://raw.githubusercontent.com/Jacoby6000/bigolchungus/feature/readme/setup-miner.sh && \
```

Comment:
wrong link

Reply:
That's intentional atm.  I'm testing to make sure all of the stuff is copy/pastable and that the setup script works.

Reply:
I'll be sure to change it back